zombienet/0004-webrtc/test-webrtc.toml

```diff
@@ -0,0 +1,30 @@
+[settings]
+enable_tracing = false
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+default_command = "substrate"
+
+chain = "gen-db"
+chain_spec_path = "zombienet/0004-webrtc/chain-spec.json"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = false
+  db_snapshot="https://storage.googleapis.com/zombienet-db-snaps/substrate/0001-basic-warp-sync/chains-0bb3f0be2ce41b5615b224215bcc8363aa0416a6.tgz"
+
+  [[relaychain.nodes]]
+  name = "bob"
+  validator = false
+  db_snapshot="https://storage.googleapis.com/zombienet-db-snaps/substrate/0001-basic-warp-sync/chains-0bb3f0be2ce41b5615b224215bcc8363aa0416a6.tgz"
+
+  #we need at least 3 nodes for warp sync
+  [[relaychain.nodes]]
+  name = "charlie"
+  validator = false
+  db_snapshot="https://storage.googleapis.com/zombienet-db-snaps/substrate/0001-basic-warp-sync/chains-0bb3f0be2ce41b5615b224215bcc8363aa0416a6.tgz"
+
+  [[relaychain.nodes]]
+  name = "dave"
+  validator = false
+  args = ["--sync warp"]
```

Comment:
Dumb question - I haven't reviewed entire PR:
How `webrtc` is enabled in this test?

Reply:
I see: by default. So proably we don't need this test at all. `0001` should cover it.

Reply:
via chain spec. I've modified bootnodes URL to `/ip4/127.0.0.1/udp/30333/webrtc/p2p/12D3KooWFvMbTsNZ8peGS8dbnRvNDBspstupzwYC9NVwbzGCLtDt`

Reply:
> [0001](https://github.com/paritytech/substrate/blob/master/zombienet/0001-basic-warp-sync/test-warp-sync.toml) should cover it.

how? the default transport is still TCP. what I need is to disable TCP.. remove all protocols from advertised addresses except webrtc?